docs/src/pages/components/date-picker/FormPropsDatePickers.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import TextField from '@material-ui/core/TextField';
+import AdapterDateFns from '@material-ui/lab/AdapterDateFns';
+import LocalizationProvider from '@material-ui/lab/LocalizationProvider';
+import DatePicker from '@material-ui/lab/DatePicker';
+
+export default function FormPropsDatePickers() {
+  const [value, setValue] = React.useState<Date | null>(null);
+
+  return (
+    <LocalizationProvider dateAdapter={AdapterDateFns}>
+      <div style={{ width: 300 }}>
```

Comment:
What do you think about using a `<Stack>` to add spacing between the items? It could display better:

<img width="279" alt="Screenshot 2021-04-30 at 11 23 04" src="https://user-images.githubusercontent.com/3165635/116675743-72c35000-a9a6-11eb-9987-562df2eda8e0.png">

The other demos seem to use `margin="normal"` on text field, I guess it's an alternative.

Reply:
Seems to work fine without introducing a new component.

Reply:
> Seems to work fine without introducing a new component. 

So, you mean using `margin="normal"`?